mmdet/integration/nncf/compression.py

```diff
@@ -78,12 +78,24 @@ def get_nncf_config_from_meta(path):
             'get_nncf_config_from_meta should be run for NNCF-compressed checkpoints only'
 
     config_text = meta['config']
-    with tempfile.NamedTemporaryFile(prefix='config_', suffix='.py', mode='w') as f_tmp:
+
+    with tempfile.NamedTemporaryFile(prefix='config_', suffix='.py',
+                                     mode='w', delete=False) as f_tmp:
         f_tmp.write(config_text)
-        cfg = mmcv.Config.fromfile(f_tmp.name)
```

Comment:
Was any issue here? Why was this line changed?

Reply:
Yes, there was an issue, and this change fixes the bug.